test/System.Web.Http.Cors.Test/CorsMessageHandlerTest.cs

```diff
@@ -7,13 +7,22 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Web.Cors;
+using System.Web.Http.ExceptionHandling;
 using System.Web.Http.Hosting;
 using Microsoft.TestCommon;
 
 namespace System.Web.Http.Cors
 {
     public class CorsMessageHandlerTest
     {
+        private class PassthroughExceptionHandler : IExceptionHandler
```

Comment:
I feels like it's not the best place for this class, however i have no idea where it should be placed(
Any suggestions would be appreciated

Reply:
Bottom of this class is fine.

Reply:
Done